.github/workflows/test.yml

```diff
@@ -506,6 +297,8 @@ jobs:
           make -C test control-plane WHAT="External Gateway"
         elif [ "${{ matrix.target }}" == "kv-live-migration" ]; then
           make -C test control-plane WHAT="Kubevirt Virtual Machines"
+        elif [ "${{ matrix.target }}" == "skip" ]; then
```

Comment:
I'm trying to come up with a very basic e2e test that would ensure that the helm deployment worked, but skips anything too fancy. Is there an existing one I could use for that purpose or should I create a new target here?

Reply:
@tssurya ^^

Reply:
So.. I think instead of making this an e2e test, we should change one specific lane from using OVNKKIND to instead using Helm charts to deploy the pieces onto a raw KIND cluster just like users would do this in prod on their bare k8s cluster.
So edit one lane to:
- install raw k8s kind
- use helm charts to install ovnkube cni on that cluster
does that make sense?